devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -226,7 +230,13 @@ private getButtonMap() {[
 				"02" : 3,
 				"03" : 1,
 				"04" : 2
-		]
+		],
+        "SceneSwitch-EM-3.0" : [
+				"01" : 1,
+                "02" : 2,
+                "03" : 3,
+                "04" : 4
+        ]
```

Comment:
The spacing should be tabs and not spaces.

Reply:
made new commit to replace spaces, please review again